src/telemetry/index.ts

```diff
@@ -89,7 +89,7 @@ export function _resetSharedProperties(): void {
 
 let telemetryReporter: TelemetryReporter | undefined;
 function getTelemetryReporter() {
-    if (!isTestExecution() && telemetryReporter) {
```

Comment:
this doesn't change anything, but the check didn't make any sense to me - Create a new reporter each time if we're testing.